config/notifiers.go

```diff
@@ -870,6 +882,77 @@ func (c *JiraConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if c.IssueType == "" {
 		return fmt.Errorf("missing issue_type in jira_config")
 	}
+	return nil
+}
+
+type RocketchatAttachmentField struct {
+	Short *bool  `json:"short"`
+	Title string `json:"title,omitempty"`
+	Value string `json:"value,omitempty"`
+}
+
+type RocketchatAttachmentActionType string
+
+type RocketchatMessageProcessingType string
+
+const (
+	ProcessingTypeSendMessage        RocketchatMessageProcessingType = "sendMessage"
+	ProcessingTypeRespondWithMessage RocketchatMessageProcessingType = "respondWithMessage"
+)
+
+type RocketchatAttachmentAction struct {
+	Type               RocketchatAttachmentActionType  `json:"type,omitempty"`
+	Text               string                          `json:"text,omitempty"`
+	URL                string                          `json:"url,omitempty"`
+	ImageURL           string                          `json:"image_url,omitempty"`
+	IsWebView          bool                            `json:"is_webview"`
+	WebviewHeightRatio string                          `json:"webview_height_ratio,omitempty"`
+	Msg                string                          `json:"msg,omitempty"`
+	MsgInChatWindow    bool                            `json:"msg_in_chat_window"`
+	MsgProcessingType  RocketchatMessageProcessingType `json:"msg_processing_type,omitempty"`
```

Comment:
```suggestion
	MsgProcessingType  string `json:"msg_processing_type,omitempty"`
```

I think we can also do the same here. If we need a type alias in future we can just add one, but I don't see a strong need for it right now.